docs/welcome.md

```diff
@@ -0,0 +1,69 @@
+# Welcome!
+
+Before you do anything else, take a picture of this document with your phone, since you might need it one day.
+
+You are now a member of Pawprint Prototyping, Santa Clara's first and only hackerspace.  Here's a few resources you should be aware of:
+
+https://hass.pawprint.space
+
+This is our Home Assistant install.  You can view the outside cameras, remotely open the door, see if the lights are on, etc.  Just log in with your LDAP credentials.  You should probably write down your LDAP account name here:
+
+
+
+
+
+https://ipa.pawprint.space (Internal only!)
+
+This is our IPA portal, which you can use to manage your LDAP account.  This only works if you're at the space.
+
+https://t.me/+06eI4qQVzXJjYzAx 
+
+This is the link to join our Telegram group, where most of the group discussion happens.
+
+(669) 696 7297
+
+You can call this number to ring the phones at the space, in case you want to talk to someone there.
```

Comment:
Perhaps worth noting: 

- If no one at the space answers, that number rings through to all of the board members (yes, even at 02:00 am.)
- A board  member can also be reached by dialing 0 from any of the VOIP phones in the space.